src/components/Layout/LayoutHeader.tsx

```diff
@@ -51,6 +51,16 @@ export const LayoutHeader = ({
     }
   };
 
+  // if our search result includes a hash to the main content anchor, remove it.
+  const transformItems = (items) => {
+    items.map((item) => {
+      if (item.url.includes('#pageMain')) {
+        item.url = item.url.replace('#pageMain', '');
+      }
+    });
+    return items;
+  };
+
```

Comment:
Should these changes be part of this PR?

Reply:
No, I'm not sure where those changes came from. Thank you.